auto3dseg/algorithm_templates/dints/scripts/infer.py

```diff
@@ -20,6 +20,7 @@
 from monai import transforms
 from monai.apps.auto3dseg.auto_runner import logger
 from monai.apps.utils import DEFAULT_FMT
+from monai.auto3dseg.utils import datafold_read
```

Comment:
need to add 
```
import torch.distributed as dist
```

Reply:
Thank you for identifying this one. The necessary correction has been made and the line has now been added.